Recipes_Technical.tex

```diff
@@ -47,50 +47,69 @@ \subsubsection{\REC*{metis_pupil_imaging}: Pupil imaging}\label{rec:metis_pupil_
+\label{rec:metiscalchophome}
+\label{rec:metis_cal_chophome}
+
+The recipe \REC{metis_cal_chophome} aims to detect chopper mirror zero positions. It is foreseen to be carried out on a daily basis; at the very least it is necessary after changes that possibly affect the absolute chopper position, e.g.\ after instrument interventions or unforeseen events like earthquakes (cf.\ Section ``Chopper Home Position'' in  \cite{METIS-calibration_plan}).
+
+The procedure consists in measuring the position of a point source
+from the \ac{WCU}, which has been centred on the \ac{WFS} pyramid (in
+the K-band), in the \CODE{IMG_LM} mode.  The recipe measures the
+position of the point source on the LM detector and derives a
+coordinate offset between the \ac{WFS} Pyramid focal plane and the
+science focal planes by comparing to the respective positional
+metrology values of the \ac{WFS} field selector and the chopper.
 %  (the latter in plural because we have already calibrated the relative astrometry between between IMG-LM and IMG-N/LMS).
 
-\begin{figure}[ht]
-  \centering
-  \includegraphics[width=0.5\textheight]{figures/metis_img_chophome_v0.83.pdf}
-  \caption[Recipe: \REC*{metis_img_chophome}]{\REC*{metis_img_chophome} --
-    Recipe workflow to detect the zero position of the chopper mirror.}
-  \label{Fig:rec_chop_home}
-\end{figure}
-
-\begin{recipedef}\label{rec:metisimgchophome}\label{rec:metis_img_chophome}
-Name:		& \REC{metis_img_chophome} \\
+\begin{recipedef}
+Name:		& \REC{metis_cal_chophome} \\
 Purpose:	& Detection of the chopper mirror home position \\
 Type:		& Calibration\\
 Requirements: & None \\
 Templates:      & \TPL{METIS_img_lm_cal_ChopperHome} \\
-Input data:     & \RAW{LM_CHOPPERHOME_RAW} \\
+Input data:     & \RAW{LM_CHOPHOME_RAW} \\
+                & \RAW{LM_WCU_OFF_RAW} \\
                 & \EXTCALIB{PERSISTENCE_MAP}  \\
                 & \STATCALIB{LINEARITY_2RG}  \\
                 & \STATCALIB{GAIN_MAP_2RG}  \\
-%                & \EXTCALIB{BADPIX_MAP_2RG}  \\
-                & \PROD{MASTER_DARK_2RG}  \\
-                & \PROD{MASTER_IMG_FLAT_LAMP_LM}  \\
+                & \EXTCALIB{BADPIX_MAP_2RG}  \\
 Matched Keywords & \FITS{DET.DIT} \\
                  & \FITS{DET.NDIT} \\
                  & \FITS{DRS.FILTER} \\
 Parameters: 	& None\\
 Algorithm:      & remove detector signature\\
                 & remove median background\\
-                & apply flatfield\\
                 & detect reference source from \ac{WCU} via centroid peak detection\\
                 & Calculate mirror offset\\
-Output data:	& Offset of the chopper mirror to be piped either into the \ac{ICS} for correction \\
-                & or to be used in the pipeline for astrometric correction\\
+Output data:	& Stacked chopperhome image \\
+                & Offset of the chopper mirror (as header keywords) to be piped either into the \ac{ICS} for correction  or to be used in the pipeline for astrometric correction\\
```

Comment:
Perhaps this is the time to actually put something sensible here?

I have explicitly hardcoded an exception for this particular recipe, and I would like to get rid of it: https://github.com/AstarVienna/METIS_DRLD/blob/90b82bf96e1ecb04ee097be7c7a73919ea44613d/codes/drld_parser/hacks.py#L194-L197

Also, currently the tests fail because the hardcoded exception is for the old name of the recipe.